presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -139,9 +159,9 @@ private void testCreatePartitionedTable(Session session, FileFormat fileFormat)
                 ", _real REAL" +
                 ", _double DOUBLE" +
                 ", _boolean BOOLEAN" +
-//                ", _decimal_short DECIMAL(3,2)" +
-//                ", _decimal_long DECIMAL(30,10)" +
-//                ", _timestamp TIMESTAMP" +
+                returnSqlIfFormatSupportsDecimalsAndTimestamps(fileFormat, ", _decimal_short DECIMAL(3,2)" +
```

Comment:
We often use the trick of `"" +` to format multi-line strings so that the arguments line up:
```java
returnSqlIfFormatSupportsDecimalsAndTimestamps(fileFormat, "" +
        ", _decimal_short DECIMAL(3,2)" +
        ", _decimal_long DECIMAL(30,10)" +
        ", _timestamp TIMESTAMP") +
```
This works with the code style and IntelliJ formatter rules.

Reply:
That's much neater; changed in all calls to the method.